spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-discovery-with-bootstrap/src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,16 @@
+spring:
```

Comment:
I've added an integration test with bootstrap enabled, that should catch these kind of things in the future.